### ISSUE TITLE
Use argument values as default values

### ISSUE THREAD

**Issue by betatim** [nteract/papermill#54]
I was half expecting the values in the cell tagged as "parameters" to work as default values. This would be convenient for things like random seeds.

**Comment by betatim** [comment]
For readers from the future: [this notebook](https://gist.github.com/betatim/f85d5f7ca475466c196489f1ddba9220) with `papermill Step1.ipynb output/Step1.ipynb -p mean 3.141 -p std 2` gives the below traceback:
```
...
  File "/Users/thead/git/nteract/papermill/papermill/execute.py", line 171, in execute_notebook
    raise_for_execution_errors(nb, output)
  File "/Users/thead/git/nteract/papermill/papermill/execute.py", line 294, in raise_for_execution_errors
    raise error
papermill.exceptions.PapermillExecutionError: 
---------------------------------------------------------------------------
Exception encountered at "In [2]":
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-2-16374ee1bb9c> in <module>()
      1 import numpy as np
      2 
----> 3 np.random.seed(int(seed))
      4 
      5 X = np.random.normal(mean, std, (100, 2))

NameError: name 'seed' is not defined
```

**Comment by rgbkrk** [comment]
I... didn't realize that didn't work. Any values that aren't set should be set as default values.

**Comment by betatim** [comment]
Closed by #74
